### PR TITLE
Added unit tests for config pkg.

### PR DESCRIPTION
**Pull request by svInfra17**
## Description
PR to include Unit Tests for config package.

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: pkg/config/config_test.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+		// Reset environment variables
+		defer unsetEnv()
+
+		// Parse config
+		config := new(config.Config)
+		err := config.Parse()
+
+		// Assert no error generated
+		It("shoud not generate error", func() {
```
Comment (caseydavenport) [inline pkg/config/config_test.go]: I think this is actually a bug - we'd like it to return an error if invalid values are given to the controller.

Comment (svInfra17) [reply pkg/config/config_test.go]: @caseydavenport , we are passing valid values in this test, so it should not generate any error.

Comment (caseydavenport) [reply pkg/config/config_test.go]: Oops yeah, you're right.

I meant to put this comment on line 126, where we're passing invalid values and it's not generating an error.

Comment (caseydavenport) [reply pkg/config/config_test.go]: Or actually, it's just the comment that's wrong.

File: pkg/config/config_test.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		setWrongEnv()
+
+		// Reset environment variables
+		defer unsetEnv()
+
+		// Parse config
+		config := new(config.Config)
+		err := config.Parse()
+
+		// Assert error is generated
```
Comment (caseydavenport) [inline pkg/config/config_test.go]: This comment is wrong.

Comment (svInfra17) [reply pkg/config/config_test.go]: @caseydavenport, fixed the spec description that it should generate an error.

**Pull request merged**